crates/re_components/src/coordinates.rs

```diff
@@ -201,6 +201,13 @@ impl ViewCoordinates {
         )
     }
 
+    /// Returns a RUB view coordinates.
+    ///
+    /// The Rerun viewer uses this as a default whenever no view coordinates are specified explicitly.
```

Comment:
We use it as a default in some narrow set of circumstances, but not always.

For 2D spaces (e.g. images) we default to X=Right, Y=Down
For 3D spaces we don't have a default up axis.

Reply:
> For 2D spaces (e.g. images) we default to X=Right, Y=Down
I codified that now as well :). But you're right, the comment is misleading